pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1436,8 +1436,11 @@ class Build {
                                             """.stripIndent(), returnStdout: true, returnStatus: false).replaceAll('\n', '')
             } else {
                 context.println "Windows detected - running bat to generate SHA256 sums in writeMetadata"
-                hash = context.bat(script: "sha256sum ${file} | cut -f1 -d' '") // .replaceAll('\n', '')
+                String hash_stdout = context.bat(script: "sha256sum ${file}", returnStdout: true, returnStatus: false)
+                // Windows batch returns stdout of <command invoked>\r\n<sha256sum output>\r\n
+                hash = hash_stdout.split('\r').last().replaceAll('\n', '').split(' ').first()
             }
+            context.println "archive sha256 = ${hash}"
```

Comment:
Is this for debug or on purpose to leave as is?

Reply:
@sophia-guo I think this is useful, otherwise the hash is not printed out for each archive type, due to the if further down that only writes the first one only...